org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/program/Program.d

```diff
@@ -23,167 +25,95 @@ import org.eclipse.swt.widgets.Event;
-            GNOME_ICON_LOOKUP_FLAGS_ALLOW_SVG_AS_THEMSELVES = 1<<2
-        }
-        enum {
-            GNOME_VFS_MAKE_URI_DIR_NONE = 0,
-            GNOME_VFS_MAKE_URI_DIR_HOMEDIR = 1 << 0,
-            GNOME_VFS_MAKE_URI_DIR_CURRENT = 1 << 1
-        }
-        alias int GnomeVFSMakeURIDirs;
-        enum {
-            GNOME_VFS_OK,
-            GNOME_VFS_ERROR_NOT_FOUND,
-            GNOME_VFS_ERROR_GENERIC,
-            GNOME_VFS_ERROR_INTERNAL,
-            GNOME_VFS_ERROR_BAD_PARAMETERS,
-            GNOME_VFS_ERROR_NOT_SUPPORTED,
-            GNOME_VFS_ERROR_IO,
-            GNOME_VFS_ERROR_CORRUPTED_DATA,
-            GNOME_VFS_ERROR_WRONG_FORMAT,
-            GNOME_VFS_ERROR_BAD_FILE,
-            GNOME_VFS_ERROR_TOO_BIG,
-            GNOME_VFS_ERROR_NO_SPACE,
-            GNOME_VFS_ERROR_READ_ONLY,
-            GNOME_VFS_ERROR_INVALID_URI,
-            GNOME_VFS_ERROR_NOT_OPEN,
-            GNOME_VFS_ERROR_INVALID_OPEN_MODE,
-            GNOME_VFS_ERROR_ACCESS_DENIED,
-            GNOME_VFS_ERROR_TOO_MANY_OPEN_FILES,
-            GNOME_VFS_ERROR_EOF,
-            GNOME_VFS_ERROR_NOT_A_DIRECTORY,
-            GNOME_VFS_ERROR_IN_PROGRESS,
-            GNOME_VFS_ERROR_INTERRUPTED,
-            GNOME_VFS_ERROR_FILE_EXISTS,
-            GNOME_VFS_ERROR_LOOP,
-            GNOME_VFS_ERROR_NOT_PERMITTED,
-            GNOME_VFS_ERROR_IS_DIRECTORY,
-            GNOME_VFS_ERROR_NO_MEMORY,
-            GNOME_VFS_ERROR_HOST_NOT_FOUND,
-            GNOME_VFS_ERROR_INVALID_HOST_NAME,
-            GNOME_VFS_ERROR_HOST_HAS_NO_ADDRESS,
-            GNOME_VFS_ERROR_LOGIN_FAILED,
-            GNOME_VFS_ERROR_CANCELLED,
-            GNOME_VFS_ERROR_DIRECTORY_BUSY,
-            GNOME_VFS_ERROR_DIRECTORY_NOT_EMPTY,
-            GNOME_VFS_ERROR_TOO_MANY_LINKS,
-            GNOME_VFS_ERROR_READ_ONLY_FILE_SYSTEM,
-            GNOME_VFS_ERROR_NOT_SAME_FILE_SYSTEM,
-            GNOME_VFS_ERROR_NAME_TOO_LONG,
-            GNOME_VFS_ERROR_SERVICE_NOT_AVAILABLE,
-            GNOME_VFS_ERROR_SERVICE_OBSOLETE,
-            GNOME_VFS_ERROR_PROTOCOL_ERROR,
-            GNOME_VFS_ERROR_NO_MASTER_BROWSER,
-            GNOME_VFS_ERROR_NO_DEFAULT,
-            GNOME_VFS_ERROR_NO_HANDLER,
-            GNOME_VFS_ERROR_PARSE,
-            GNOME_VFS_ERROR_LAUNCH,
-            GNOME_VFS_ERROR_TIMEOUT,
-            GNOME_VFS_ERROR_NAMESERVER,
-            GNOME_VFS_ERROR_LOCKED,
-            GNOME_VFS_ERROR_DEPRECATED_FUNCTION,
-            GNOME_VFS_ERROR_INVALID_FILENAME,
-            GNOME_VFS_ERROR_NOT_A_SYMBOLIC_LINK,
-            GNOME_VFS_NUM_ERRORS
-        }
-        alias int GnomeVFSResult;
-
-        enum {
-            GNOME_VFS_MIME_APPLICATION_ARGUMENT_TYPE_URIS,
-            GNOME_VFS_MIME_APPLICATION_ARGUMENT_TYPE_PATHS,
-            GNOME_VFS_MIME_APPLICATION_ARGUMENT_TYPE_URIS_FOR_NON_FILES
-        }
+// DWT TODO: move this and GIO into OS (though seems to be GTK in newer versions)
```

Comment:
Why not move this now?

Reply:
I've pushed a new (temporary) commit with some changes, but I'll point out some parts I would like you to check before I continued.